progenitor-macro/src/lib.rs

```diff
@@ -174,7 +179,14 @@ fn do_generate_api(item: TokenStream) -> Result<TokenStream, syn::Error> {
             patch,
             convert,
         } = serde_tokenstream::from_tokenstream(&item.into())?;
+
+        let deps: HashMap<String, String> = built_info::DEPENDENCIES
+            .iter()
+            .map(|(name, version)| (name.to_string(), version.to_string()))
+            .collect();
+
```

Comment:
why is this useful for the macro form?

Reply:
yea, the dependencies are not used by the macro. happy to remove.
alternatively we could emit a dependency block that is used by tools like https://github.com/reitermarkus/cargo-eval and others.
```
//! ```cargo
//! ...
//! ```
```